extensions-core/druid-bloom-filter/src/test/java/org/apache/druid/query/filter/sql/BloomDimFilterSqlTest.java

```diff
@@ -275,6 +276,7 @@ public void testBloomFilters() throws Exception
     );
   }
 
+  @Ignore
```

Comment:
Can you add an explanation of why it's ignored (the annotation takes a value, which can be used for the comment).